packages/remix-dev/__tests__/flat-routes-test.ts

```diff
@@ -289,6 +289,16 @@ describe("flatRoutes", () => {
         },
       ],
 
+      // validate windows path
```

Comment:
all tests already replace '/' with path.sep https://github.com/remix-run/remix/blob/5f30962cdf2d7642eae24af8a392a1d136bf641d/packages/remix-dev/__tests__/flat-routes-test.ts#L615, but i wanted to explicitly test it :)